BlueprintUICommonControls/Sources/Image.swift

```diff
@@ -37,6 +43,10 @@ public struct Image: Element {
             config[\.contentMode] = contentMode.uiViewContentMode
             config[\.layer.minificationFilter] = .trilinear
             config[\.tintColor] = tintColor
+            if blockAccessibilityDiscription {
+                // Seting `isAccessibilityElement = false` isn't enough here, VoiceOver is very aggressive in finding images to discribe. We need to explicitly remove the `.image` trait.
+                config[\.accessibilityTraits] = UIAccessibilityTraits.none
```

Comment:
i wish i could do `.remove(.image)` here but that's not how this works, in this case its fine because its backed by a native `UIImageView` which has `.image` as its only default trait.